Managers/MovableMan.h

```diff
@@ -295,14 +295,14 @@ class MovableMan : public Singleton<MovableMan>, public Serializable {
 // Arguments:       Which team to try to get an Actor for. 0 means first team, 1 means 2nd.
 //                  The Scene point to search for the closest to.
 //                  The maximum radius around that scene point to search.
-//                  A float to be filled out with the distance of the returned closest to
+//                  A vector to be filled out with the distance of the returned closest to
```

Comment:
I haven't used it before, but it makes all the sense to me to include the distance as a vector rather than its' magnitude, as it save's one extra step if the modder wants to access the distance as a vector. As I mentioned, this function's lua access was previously broken and doesn't exist anywhere in data.

Reply:
Old comments from data shouldn't be your bar for correctness lol, We capitalize class names in method comments, and Vector is a class. Feel free to change it on GetClosestEnemyActor too, if that's what bothers you, I've not problem with that.

As for the rest of this, fair enough. If it was never lua-usable then it doesn't really matter if it's changed haha, as long as it all works fine.